### PR TITLE
update mpifileutils to commit 49b04c1

### PR DESCRIPTION
**Pull request by dsikich**
update mpifileutils to commit 49b04c1
Signed-off-by: Danielle Sikich <danielle.sikich@intel.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 NAME      := mpifileutils
 SRC_EXT   := gz
 REPO_NAME := mpifileutils-pkg
-GIT_SHORT := 49b04c1
+PKG_GIT_COMMIT := 49b04c1
+GITHUB_PROJECT := hpc/$(NAME)
```
Comment (brianjmurrell) [inline Makefile]: If you have a need to commit again, it would be nice to line up these `:=`.

**Comment by brianjmurrell** [comment]
https://github.com/daos-stack/packaging/'s master has just been updated.  You should update your packaging/ here from it.

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,9 @@
 NAME      := mpifileutils
 SRC_EXT   := gz
 REPO_NAME := mpifileutils-pkg
-GIT_SHORT := bdc379b
+GIT_SHORT := 49b04c1
```
Comment (brianjmurrell) [inline Makefile:4]: This is not used any more, once you update packaging/ again.

File: Makefile [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+PKG_GIT_COMMIT := 49b04c1
+GITHUB_PROJECT := hpc/$(NAME)
 
 BUILD_DEFINES := --define "git_short $(GIT_SHORT)"
```
Comment (brianjmurrell) [inline Makefile:8]: This should be removed from this file.

Comment (brianjmurrell) [reply Makefile:6]: Please don't resolve conversations that I start.

File: mpifileutils.spec [hunk 4]
```diff
@@ -58,7 +58,6 @@ Source:		https://github.com/hpc/%{name}/archive/v%{version}.tar.gz
 %if %{defined git_short}
 Patch1:		v%{version}..%{?git_short}.patch
 %endif
```
Comment (brianjmurrell) [inline mpifileutils.spec:60]:
```suggestion
%if "%{?commit}" != ""
Patch1: %{version}..%{commit}.patch
%endif
```

File: v0.11..49b04c1.patch [hunk 5]
```diff
@@ -1,3 +1,105 @@
+diff --git a/.github/CONTRIBUTING.md b/.github/CONTRIBUTING.md
```
Comment (brianjmurrell) [inline v0.11..49b04c1.patch:1]: Drop the leading `v` from this patch so that it's named `0.11..49b04c1.patch`.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
There are a number of changes that are still required to get this to build.  Please make them and then we can go from there if it's still not building.

File: Makefile (hunk 3 above)
Comment (dsikich) [reply Makefile:6]: @brianjmurrell I had it removed yesterday and the build was failing. can try again later but as soon as I removed these suggestions the build passed.

**Comment by daltonbohning** [comment]
This PR is stale (opened Nov 2021), Danielle is no longer on the team, and we have a more recent #27.
@brianjmurrell Any objection to me closing this PR?

**Comment by brianjmurrell** [comment]
Not at all.

**Pull request closed without merging**